specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -803,6 +912,7 @@ model DimensionFilter {
 }
```

Comment:
are there specific list of dimension names supported

Reply:
Yes currently we support {SamplerName, EngineId} as dimension name.